config/templates/cloud-config-controller

```diff
@@ -56,14 +79,21 @@ coreos:
       drop-ins:
         - name: 10-etcd.conf
           content: |
+            [Unit]
+            Requires=cfn-etcd-environment.service
```

Comment:
This should be changed according to how you'd approach for https://github.com/coreos/kube-aws/pull/226/files#r96115270.
* Making `cfn-etcd-environment.service` to have `Type=simple` or `Type=oneshot` without implementing retries in bash while having `Requires=cfn-etcd-environment.service` here would permanently keep `flanneld.service` failed once `cfn-etcd-environment.service` failed.
   * So I'd suggest either (1) making `cfn-etcd-environment.service` a `Type=simple` retried by systemd while making `flanneld.service` not to have `Requires=cfn-etcd-environment.service` but to have `Wants=cfn-etcd-environment.service` + `After=cfn-etcd-environment.service` + `ExecStartPre=/usr/bin/systemctl is-active cfn-etcd-environment.service` or (2) making `cfn-etcd-environment.service` a `Type=simple` or `Type=oneshot` retried in bash. AFAIK either (1) or (2) allows `flanneld` to be periodically retried until `cfn-etcd-environment` succeeds. (1) is recommended just because it is used all over the places in `cloud-config-*`.